src/js/scaleworkshop.js

```diff
@@ -71,7 +71,7 @@ var current_approximations = {
     denominator_limits: [], // the prime limit of each denominator
     ratio_limits: [] // the prime limit of each ratio
 }
-var prime_counter = [0, 10];
+var prime_counter = [0, 10]; // used for the prime number boxes in Approximate
```

Comment:
Why not rename the variable to something, which contains all the information in the comment, so that a comment is unnecessary? For example `primeCounterForTheBoxesInApproximate`?